[debug] [2018-12-11T17:56:41.448Z] ----------------------------------------------------------------------
[debug] [2018-12-11T17:56:41.450Z] Command:       /usr/bin/node /usr/local/bin/firebase serve --only functions,hosting
[debug] [2018-12-11T17:56:41.450Z] CLI Version:   6.1.1
[debug] [2018-12-11T17:56:41.450Z] Platform:      linux
[debug] [2018-12-11T17:56:41.450Z] Node Version:  v8.10.0
[debug] [2018-12-11T17:56:41.451Z] Time:          Tue Dec 11 2018 17:56:41 GMT+0000 (WET)
[debug] [2018-12-11T17:56:41.451Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-12-11T17:56:41.463Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-12-11T17:56:41.463Z] > authorizing via signed-in user
[debug] [2018-12-11T17:56:41.463Z] [iam] checking project tysix-75b86 for permissions ["firebase.projects.get"]
[debug] [2018-12-11T17:56:41.465Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/tysix-75b86:testIamPermissions  

[debug] [2018-12-11T17:56:42.440Z] <<< HTTP RESPONSE 200
[debug] [2018-12-11T17:56:42.448Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/tysix-75b86  
 
[debug] [2018-12-11T17:56:43.312Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/home/diogo/www/tysix'...
[info] 
[debug] [2018-12-11T17:56:48.756Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:102:23)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:13:26)
    at Command.module.exports.Command.description.option.option.option.option.before.action [as _action] (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:55:12)
[error] 
[error] Error: An unexpected error has occurred.
